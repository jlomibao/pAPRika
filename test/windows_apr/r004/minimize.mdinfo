

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1130E+02     4.6894E-01     1.5986E+00     H14        51

 BOND    =        1.1109  ANGLE   =      107.7368  DIHED      =       51.4531
 VDWAALS =      -46.2360  EEL     =     1337.5546  EGB        =     -135.9308
 1-4 VDW =        5.2067  1-4 EEL =    -2132.2017  RESTRAINT  =        0.0030
 EAMBER  =     -811.3063
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================
