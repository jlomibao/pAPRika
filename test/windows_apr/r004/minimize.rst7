defa
  125
  -0.5217284   4.9759806   1.8352680   1.0157632   4.9108522   1.8351961
  -0.8855188   4.2791163   0.6084415  -0.8854167   4.2789182   3.0620147
  -0.9016902   6.0041172   1.8353610   1.4813187   5.9031697   1.8352518
   1.3195448   4.1855843   0.6086080   1.3196407   4.1853825   3.0616417
   2.6868532   4.0833383   0.1137412   2.6869792   4.0831303   3.5564081
   0.2013724   3.8584536   3.7830146   0.2012135   3.8588130  -0.1127381
   0.1802941   3.3487410   4.8903505   0.1800435   3.3493645  -1.2201770
   2.6633055   4.0469876  -0.9815055   3.2459756   4.9656358   0.4459596
   2.6634989   4.0466460   4.6516518   3.2460622   4.9654796   3.2242608
  -4.6359934   1.9609913   1.8353773  -3.8101693   3.2595658   1.8353641
  -4.2146566   1.2971937   0.6086163  -4.2146419   1.2971714   3.0621287
  -5.7162748   2.1465392   1.8353845  -4.4363399   4.1592026   1.8353926
  -3.0303704   3.1595353   0.6085639  -3.0303377   3.1594919   3.0621567
  -2.2566218   4.2914759   0.1136524  -2.2564488   4.2913670   3.5569935
  -3.3067564   2.0275268   3.7832422  -3.3068145   2.0275818  -0.1125192
  -2.8757734   1.7539113   4.8904230  -2.8759270   1.7540273  -1.2197340
  -2.2364047   4.2526307  -0.9815752  -2.7400175   5.2175482   0.4455481
  -2.2360812   4.2525075   4.6522178  -2.7398260   5.2174773   3.2251773
  -4.0834560  -3.1098999   1.8352277  -4.7950375  -1.7453608   1.8352859
  -3.2979905  -3.0769695   0.6084176  -3.2978104  -3.0769769   3.0619265
  -4.7843474  -3.9526259   1.8352715  -5.8872407  -1.8377084   1.8353299
  -4.3184600  -1.1200700   0.6084860  -4.3183353  -1.1201022   3.0620563
  -4.9114476   0.1162890   0.1138218  -4.9113614   0.1161893   3.5568531
  -3.4762400  -1.9255439   3.7830698  -3.4765002  -1.9255200  -0.1126716
  -3.0236745  -1.6891540   4.8902108  -3.0241256  -1.6891344  -1.2198760
  -4.8678336   0.1144974  -0.9814114  -5.9550800   0.1610909   0.4458763
  -4.8677270   0.1142975   4.6520854  -5.9550054   0.1609725   3.2248326
   0.5850518  -5.1654770   1.8348843  -0.9525007  -5.1002337   1.8349934
   0.9488677  -4.4684750   0.6081158   0.9490200  -4.4685770   3.0616643
   0.9649517  -6.1936365   1.8348091  -1.4181206  -6.0925210   1.8349918
  -1.2561425  -4.3749351   0.6082574  -1.2559818  -4.3749927   3.0618093
  -2.6234541  -4.2707883   0.1137612  -2.6232327  -4.2708199   3.5564748
  -0.1375602  -4.0479631   3.7828460  -0.1378164  -4.0478598  -0.1129078
  -0.1162280  -3.5379017   4.8900220  -0.1166366  -3.5378011  -1.2200734
  -2.6002639  -4.2341488  -0.9814835  -3.1837755  -5.1522792   0.4461010
  -2.5999054  -4.2341972   4.6517171  -3.1836170  -5.1522935   3.2241953
   4.6985643  -2.1491150   1.8347268   3.8734280  -3.4481293   1.8347315
   4.2768109  -1.4854504   0.6080355   4.2769048  -1.4855151   3.0614829
   5.7789409  -2.3341075   1.8346775   4.5000430  -4.3474567   1.8346860
   3.0934964  -3.3484256   0.6079926   3.0935900  -3.3484764   3.0615352
   2.3199619  -4.4806095   0.1132943   2.3201838  -4.4807249   3.5562717
   3.3694530  -2.2163773   3.7826298   3.3693450  -2.2163152  -0.1130909
   2.9384336  -1.9430252   4.8898602   2.9383223  -1.9429682  -1.2203063
   2.2996729  -4.4419045  -0.9819370   2.8035494  -5.4065443   0.4452941
   2.3000737  -4.4421225   4.6515099   2.8037431  -5.4066131   3.2241008
   4.1487975   2.9227318   1.8349500   4.8584199   1.5570331   1.8348454
   3.3640838   2.8906191   0.6075461   3.3641854   2.8904813   3.0624196
   4.8510148   3.7643534   1.8349623   5.9507571   1.6477824   1.8347898
   4.3808273   0.9320330   0.6082337   4.3809474   0.9319243   3.0614518
   4.9732732  -0.3044188   0.1130552   4.9734593  -0.3045502   3.5564671
   3.5393936   1.7378999   3.7824765   3.5392574   1.7381185  -0.1126557
   3.0853167   1.5010975   4.8889109   3.0851715   1.5014746  -1.2191043
   4.9290517  -0.3024394  -0.9821534   6.0171140  -0.3493631   0.4444347
   4.9293917  -0.3026517   4.6516820   6.0172509  -0.3494804   3.2249306
   0.0002586   0.0058384  17.9523396  -0.9138156  -0.5729800  17.7717053
   0.8630588  -0.6469389  17.7716505   0.0343175   0.8246961  17.2232931
   0.0234248   0.5614614  19.3896831   0.9307858   1.1617555  19.5294430
  -0.0232157  -0.5612607  20.4482508  -0.8309570   1.2350108  19.5295135
  -0.9305191  -1.1615839  20.3082418   0.8311850  -1.2347523  20.3082577
  -0.0001672  -0.0059424  21.8857879  -0.8630227   0.6467870  22.0664102
  -0.0342358  -0.8249879  22.6146247   0.9138748   0.5729191  22.0664566
  -0.0000862   0.0001862  -6.0013678  -0.0001807  -0.0000373  -9.0000704
   0.0001726   2.2000035 -11.2000237
