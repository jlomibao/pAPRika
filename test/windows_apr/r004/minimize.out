
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:53:19

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/r004
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   331771 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9698E+02     1.0074E+01     2.4798E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -45.6645  EEL     =     1326.5993  EGB        =     -129.6177
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -796.9808
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1130E+02     4.6902E-01     1.5985E+00     H14        51

 BOND    =        1.1092  ANGLE   =      107.7359  DIHED      =       51.4553
 VDWAALS =      -46.2316  EEL     =     1337.5028  EGB        =     -135.9177
 1-4 VDW =        5.2044  1-4 EEL =    -2132.1646  RESTRAINT  =        0.0030
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1130E+02     4.6901E-01     1.5986E+00     H14        51

 BOND    =        1.1094  ANGLE   =      107.7360  DIHED      =       51.4550
 VDWAALS =      -46.2321  EEL     =     1337.5090  EGB        =     -135.9193
 1-4 VDW =        5.2047  1-4 EEL =    -2132.1691  RESTRAINT  =        0.0030
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1130E+02     4.6900E-01     1.5986E+00     H14        51

 BOND    =        1.1096  ANGLE   =      107.7361  DIHED      =       51.4548
 VDWAALS =      -46.2326  EEL     =     1337.5147  EGB        =     -135.9207
 1-4 VDW =        5.2049  1-4 EEL =    -2132.1732  RESTRAINT  =        0.0030
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1130E+02     4.6899E-01     1.5986E+00     H14        51

 BOND    =        1.1098  ANGLE   =      107.7362  DIHED      =       51.4545
 VDWAALS =      -46.2331  EEL     =     1337.5204  EGB        =     -135.9221
 1-4 VDW =        5.2052  1-4 EEL =    -2132.1773  RESTRAINT  =        0.0030
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1130E+02     4.6898E-01     1.5986E+00     H14        51

 BOND    =        1.1100  ANGLE   =      107.7363  DIHED      =       51.4543
 VDWAALS =      -46.2336  EEL     =     1337.5263  EGB        =     -135.9236
 1-4 VDW =        5.2054  1-4 EEL =    -2132.1816  RESTRAINT  =        0.0030
 EAMBER  =     -811.3065
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1130E+02     4.6898E-01     1.5986E+00     H14        51

 BOND    =        1.1102  ANGLE   =      107.7364  DIHED      =       51.4540
 VDWAALS =      -46.2341  EEL     =     1337.5319  EGB        =     -135.9251
 1-4 VDW =        5.2057  1-4 EEL =    -2132.1854  RESTRAINT  =        0.0030
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1130E+02     4.6897E-01     1.5986E+00     H14        51

 BOND    =        1.1103  ANGLE   =      107.7365  DIHED      =       51.4538
 VDWAALS =      -46.2346  EEL     =     1337.5375  EGB        =     -135.9265
 1-4 VDW =        5.2060  1-4 EEL =    -2132.1894  RESTRAINT  =        0.0030
 EAMBER  =     -811.3063
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1130E+02     4.6895E-01     1.5986E+00     H14        51

 BOND    =        1.1106  ANGLE   =      107.7366  DIHED      =       51.4535
 VDWAALS =      -46.2351  EEL     =     1337.5435  EGB        =     -135.9280
 1-4 VDW =        5.2062  1-4 EEL =    -2132.1938  RESTRAINT  =        0.0030
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1130E+02     4.6895E-01     1.5986E+00     H14        51

 BOND    =        1.1107  ANGLE   =      107.7367  DIHED      =       51.4533
 VDWAALS =      -46.2355  EEL     =     1337.5491  EGB        =     -135.9293
 1-4 VDW =        5.2065  1-4 EEL =    -2132.1978  RESTRAINT  =        0.0030
 EAMBER  =     -811.3064
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1130E+02     4.6894E-01     1.5986E+00     H14        51

 BOND    =        1.1109  ANGLE   =      107.7368  DIHED      =       51.4531
 VDWAALS =      -46.2360  EEL     =     1337.5546  EGB        =     -135.9308
 1-4 VDW =        5.2067  1-4 EEL =    -2132.2017  RESTRAINT  =        0.0030
 EAMBER  =     -811.3063
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1130E+02     4.6894E-01     1.5986E+00     H14        51

 BOND    =        1.1109  ANGLE   =      107.7368  DIHED      =       51.4531
 VDWAALS =      -46.2360  EEL     =     1337.5546  EGB        =     -135.9308
 1-4 VDW =        5.2067  1-4 EEL =    -2132.2017  RESTRAINT  =        0.0030
 EAMBER  =     -811.3063
 NMR restraints: Bond =    0.003   Angle =     0.000   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.303    9.302    0.002    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.390  147.398    0.008    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.472   49.483    0.011    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.003   62.003    0.000    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.190 -146.178    0.012    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.545  -72.548    0.004    0.000 t
  DUM  DM1    3 --  C    BUT    2:   23.954   24.000    0.046    0.003 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.991  180.000    0.009    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.815  180.000    0.185    0.000 a
                                       Total distance penalty:      0.003
                                       Total angle    penalty:      0.000
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.472
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.16 (21.70% of Gen B)
|                Calc gb diag               3.86 (38.80% of Gen B)
|                Calc gb off-diag           3.93 (39.48% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time              9.94 (100.0% of Nonbo)
|             IPS excludes               0.00 ( 0.01% of Nonbo)
|          Nonbond force              9.94 (90.36% of Force)
|          Bond/Angle/Dihedral        1.03 ( 9.33% of Force)
|          Other                      0.03 ( 0.31% of Force)
|       Force time                11.00 (100.0% of Runmd)
|    Runmd Time                11.00 (99.80% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.02 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:53:19.328  on 04/10/2019
|           Setup done at 18:53:19.333  on 04/10/2019
|           Run   done at 18:53:30.352  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
